dts-generator/src/main/java/com/telerik/dts/DtsApi.java

```diff
@@ -441,6 +445,8 @@ private void generateInterfaceConstructorCommentBlock(JavaClass classInterface,
     private void processMethod(Method m, JavaClass clazz, boolean isInterface, Set<String> methodsSet) {
         String name = m.getName();
 
+        if (name.startsWith("zz")) return;
```

Comment:
add a comment explaining why this is necessary or link to an issue should be sufficient